lib/temperature/index.js

```diff
@@ -83,7 +87,7 @@ TemperatureSensor.prototype.start = function start() {
       if ((this.options.controller === 'bmp085') || (this.options.controller === 'bmp180')) {
         this.sensor = new BMP085(this.options.sensor);
       } else {
-        throw new Error("TODO: unsupported controller:" + this.options.controller);
```

Comment:
keep this line here and add test controller==='simulator' like:

https://github.com/rzr/generic-sensors-lite/blob/master/lib/ambientlight/index.js#L85